QueAns/views.py

```diff
@@ -0,0 +1,47 @@
+from django.http import HttpResponse
+from .models import Voting,Meetings,Satisfaction
+from datetime import datetime
+
+
+def upvote(request,user_id,question_id):
+    if request.method == "POST":
```

Comment:
No, authentication. What if someone from outside get's link(URL) to this and start DDoSing us?